test/controllers/match.test.ts

```diff
@@ -0,0 +1,197 @@
+import request from 'supertest'
+import connection from '../db-helper'
+import app from '../../src/app'
+
+process.on('uncaughtException', function (err) {
+  console.log('oh no', err);
+});
+
+const creatingSkills = async () => {
+  await request(app).post('/skills/create').send({ name: 'javascript' })
+  await request(app).post('/skills/create').send({ name: 'reactjs' })
+  await request(app).post('/skills/create').send({ name: 'typescript' })
+  await request(app).post('/skills/create').send({ name: 'react' })
+  await request(app).post('/skills/create').send({ name: 'nodejs' })
+  await request(app).post('/skills/create').send({ name: 'docker' })
+  await request(app).post('/skills/create').send({ name: 'aws' })
+  await request(app).post('/skills/create').send({ name: 'gcp' })
+  await request(app).post('/skills/create').send({ name: 'iac' })
+}
+
+const jobPost1Skills = async () => {
+  await request(app).get('/skills/1').send({ name: 'javascript' })
+  await request(app).get('/skills/2').send({ name: 'reactjs' })
+  await request(app).get('/skills/3').send({ name: 'nodejs' })
+}
+
+const jobPost2Skills = async () => {
+  await request(app).get('/skills/1').send({ name: 'javascript' })
+  await request(app).get('/skills/2').send({ name: 'reactjs' })
+  await request(app).get('/skills/3').send({ name: 'typescript' })
+  await request(app).get('/skills/4').send({ name: 'react' })
+  await request(app).get('/skills/5').send({ name: 'nodejs' })
+  await request(app).get('/skills/6').send({ name: 'docker' })
+}
+
+const jobPost3Skills = async () => {
+  await request(app).get('/skills/7')
+  await request(app).get('/skills/8')
+  await request(app).get('/skills/9')
+}
+
+const jobSeekerSkills = async () => {
+  return await request(app).get('/skills')
+}
+
+const jobSeeker = {
+  info: {
+    firstName: 'duy',
+    lastName: 'nguyen',
+    contact: 1234,
+    relocate: true,
+    seniority: 'mid',
+    startingDate: '10/12/2020',
+  },
+  credential: {
+    email: 'abc@gmail.com',
+    password: 'password',
+  },
+  skills: jobSeekerSkills()
+}
+
+const jobPost_1 = {
+  title: 'Fullstack React- & Node.js Developer',
+  jobDescription: 'We create and operate the online shops of Klamotten. Your job is to participate in the further development of our existing shop system platform',
+  seniority: 'Junior',
+  createdAt: "01/12/2020",
+  requiredSkills: jobPost1Skills()  
```

Comment:
This is not how you save relation in typeorm. And the create job post controller should create and save the relation for you